azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/PartitionContext.java

```diff
@@ -100,7 +100,7 @@ void setLease(Lease lease)
 
     void setOffsetAndSequenceNumber(EventData event)
     {
-		if (sequenceNumber >= this.sequenceNumber)
```

Comment:
original condition looks incorrect, i.e. comparing sequenceNumber to itself. It looks from setOffsetAndSequenceNumber, offset and sequenceNumber to be set from EventData